src/librawrtc/ice_gatherer.c

```diff
@@ -1454,6 +1457,11 @@ static bool interface_handler(
         return false; // Continue gathering
     }
 
+    // Ignore 0.0.0.0
+    if(sa_af(address) == AF_INET && address->u.in.sin_addr.s_addr == 0) {
```

Comment:
Let's use `sa_is_any` for that purpose which also includes IPv6 and move the code above the loopback check.